public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -293,6 +301,15 @@ export default class CloudWatchDatasource {
       return this.getEc2InstanceAttribute(region, targetAttributeName, filterJson);
     }
 
+
+    const resourceARNsQuery = query.match(/^resource_arns\(([^,]+?),\s?([^,]+?),\s?(.+?)\)/);
```

Comment:
Would like a test of this. There's existing tests for metricFindQuery in datasource.test.ts so should be quite trivial to implement.

Reply:
Added a test using metricFindQuery in 3729660f5